docs/tutorials/1_the_ibmq_account.ipynb

```diff
@@ -76,7 +76,7 @@
     "The `IBMQ` account has functions for handling administrative tasks. The credentials can be saved to disk, or used in a session and never saved.\n",
     "\n",
     "- `enable_account(TOKEN)`: Enable your account in the current session.\n",
-    "- `save_account(TOKEN)`: Save your account to disk for future use.\n",
+    "- `save_account(TOKEN, HUB, GROUP, PROJECT)`: Save your account to disk for future use and optionally specify a default provider to use when loading your account.\n",
```

Comment:
Why are HUB/GROUP/PROJECT mentioned for `save_account()` but not `enable_account()`?